[2023-03-17T19:13:16.164+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: get_file_from_endpoint.insert_values_parking_measurements manual__2023-03-17T19:12:58.024588+00:00 [queued]>
[2023-03-17T19:13:16.177+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: get_file_from_endpoint.insert_values_parking_measurements manual__2023-03-17T19:12:58.024588+00:00 [queued]>
[2023-03-17T19:13:16.177+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-17T19:13:16.177+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-17T19:13:16.177+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-17T19:13:16.192+0000] {taskinstance.py:1300} INFO - Executing <Task(MySqlOperator): insert_values_parking_measurements> on 2023-03-17 19:12:58.024588+00:00
[2023-03-17T19:13:16.197+0000] {standard_task_runner.py:55} INFO - Started process 8270 to run task
[2023-03-17T19:13:16.203+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_file_from_endpoint', 'insert_values_parking_measurements', 'manual__2023-03-17T19:12:58.024588+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmp_t6vfk0c']
[2023-03-17T19:13:16.205+0000] {standard_task_runner.py:83} INFO - Job 263: Subtask insert_values_parking_measurements
[2023-03-17T19:13:16.296+0000] {task_command.py:388} INFO - Running <TaskInstance: get_file_from_endpoint.insert_values_parking_measurements manual__2023-03-17T19:12:58.024588+00:00 [running]> on host diploma-vm.europe-west1-b.c.diploma-380314.internal
[2023-03-17T19:13:16.382+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_file_from_endpoint
AIRFLOW_CTX_TASK_ID=insert_values_parking_measurements
AIRFLOW_CTX_EXECUTION_DATE=2023-03-17T19:12:58.024588+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-17T19:12:58.024588+00:00
[2023-03-17T19:13:16.383+0000] {sql.py:254} INFO - Executing: INSERT INTO pre_parking_measurements VALUES ('0','{"error_message": "Unauthorized. Failed to authenticate user.", "error_status": 401}',' 0 ','2023-03-17 19:13:16.003440','1');
[2023-03-17T19:13:16.396+0000] {base.py:73} INFO - Using connection ID 'mysql-db' for task execution.
[2023-03-17T19:13:16.484+0000] {base.py:73} INFO - Using connection ID 'mysql-db' for task execution.
[2023-03-17T19:13:16.496+0000] {sql.py:375} INFO - Running statement: INSERT INTO pre_parking_measurements VALUES ('0','{"error_message": "Unauthorized. Failed to authenticate user.", "error_status": 401}',' 0 ','2023-03-17 19:13:16.003440','1');, parameters: None
[2023-03-17T19:13:16.497+0000] {sql.py:384} INFO - Rows affected: 1
[2023-03-17T19:13:16.516+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=get_file_from_endpoint, task_id=insert_values_parking_measurements, execution_date=20230317T191258, start_date=20230317T191316, end_date=20230317T191316
[2023-03-17T19:13:16.536+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-17T19:13:16.559+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
